.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I usually git ignore all IDE files, do you want to do that with all .idea/* files?

Reply:
Here's JetBrain's [guidance](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) on the topic.